designer/apps/pages/models.py

```diff
@@ -13,6 +17,22 @@
 
 from designer.apps.branding.models import Branding
 
+DEFAULT_DESCRIPTION = re.sub(
```

Comment:
I should have clarified, I meant for this to be a constant on the class. Since it's very class specific, it should stay in that scope.